desktop/libs/hadoop/src/hadoop/fs/hadoopfs.py

```diff
@@ -188,7 +188,12 @@ def create_home_dir(self, home_path=None):
           self.setuser(self.superuser)
           self.mkdir(home_path)
           self.chmod(home_path, mode)
-          self.chown(home_path, user, user)
+          self.chown(home_path, user)
+          try:  # Handle the case when there is no group with the same name as the user.
+            self.chown(home_path, group=user)
+          except IOError:
```

Comment:
Also log the error message? (via another str(e) or LOG.exception)

Reply:
Yeah, makes sense. Thank you!
I've just commited this change.